hbw/app/models/hbw/camunda/adapter.rb

```diff
@@ -16,6 +16,11 @@ def active_process_instances(entity_code, entity_class)
         response.status == 200 && response.body || []
       end
 
+      def delete(id)
```

Comment:
Maybe `cancel_process` or `delete_process`? Current name is too abstract I think (`Camunda::Adapter#delete`).